src/popup/locales/es.json

```diff
@@ -216,12 +283,12 @@
     "account": {
       "heading": "Cuenta",
       "accountName": "Nombre de la cuenta",
-      "copy": "copiar",
-      "send": "español",
+      "copy": "Copiar"
```

Comment:
`,` is missed here

Reply:
Strange, the tests passed.

Reply:
In all .json originals (superhero wallet and in-app ones) there is no "," in the last code line before "}".
example:
    "TipCommentsView": {
      "NoResultsMsg": "There are no comments at the moment.",
      "Reply": "Reply",
      "AddReply": "Add reply",
      "LatestReplies": "Latest Replies"
    },
    "SingleCommentView": {
      "NoResultsMsg": "There are no comments to display"
    },

As it is showed there the code lines before "}" have no ","

Reply:
> In all .json originals (superhero wallet and in-app ones) there is no "," in the last code line before "}".


It is true, is required by JSON format. But in this case the `,` is missed between the fields, like:
```
  ...
  "fieldName1": "fieldValue1"
  "fieldName2": "fieldValue2"
}
```
it should be
```
  ...
  "fieldName1": "fieldValue1",
  "fieldName2": "fieldValue2"
}
```

Also, GitHub highlights this typo with the red color:

![Screenshot 2020-08-24 at 15 29 14](https://user-images.githubusercontent.com/9007851/91007184-a59a9480-e61e-11ea-908d-3693a2ccb4d3.png)

But don't worry, anyway, we have fixed it in #476